<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testResult type="org.overture.vdmj.typechecker.tests.external.NewModuleTestCase">
<message column="1" line="18" message="Recursive function has no measure" messageType="warning" number="5012" resource="recfun-32.vdm.result"/>
<message column="12" line="19" message="Definition 'h' not used" messageType="warning" number="5000" resource="recfun-32.vdm.result"/>
<message column="1" line="24" message="Recursive function has no measure" messageType="warning" number="5012" resource="recfun-32.vdm.result"/>
<message column="1" line="24" message="Function returns unexpected type" messageType="error" number="3018" resource="recfun-32.vdm.result"/>
<result/>
</testResult>
